cfme/tests/configure/test_vmware_console_settings.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.configure.configuration import VMwareConsoleSupport
+from utils.appliance.implementations.ui import navigate_to
+from utils.version import current_version
+
+
+@pytest.mark.tier(3)
+@pytest.mark.sauce
+@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+def test_vmware_console_support(appliance):
+    """Tests that the VMware Console Support setting may be changed."""
+    navigate_to(appliance.server, 'Server')
+
+    console_type_widget = VMwareConsoleSupport.vmware_console_form.console_type
+    old_vm_console_type = console_type_widget.first_selected_option_text
+    assert old_vm_console_type, "The default VMware console type should not be empty"
+
+    for new_vm_console_type in VMwareConsoleSupport.CONSOLE_TYPES:
+        console_settings = VMwareConsoleSupport(
+            appliance=appliance,
+            console_type=new_vm_console_type
+        )
+        console_settings.update()
+        console_type_widget = VMwareConsoleSupport.vmware_console_form.console_type
+        cur_vm_console_type = console_type_widget.first_selected_option_text
+        assert cur_vm_console_type == new_vm_console_type
+
+    console_settings = VMwareConsoleSupport(
```

Comment:
REQUIRED: 
1. this is teardown code. so, it should be done by all means. 
2. please consider moving VMwareConsoleSupport object creation to regular or yield fixture.